WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -80,13 +86,16 @@ private extension TotalsView {
                     .redacted(reason: redacted ? [.placeholder] : [])
             }
             .foregroundColor(Color.primaryText)
+            .matchedGeometryEffect(id: "subtotalFieldView:_\(title)", in: totalsFieldAnimation)
+            .animation(.default, value: totalsViewModel.isShimmering)
```

Comment:
We use the `shimmeringActive` parameter to decide whether to use the placeholder, and nothing else in this function currently relies directly on the view model. Perhaps it's best to use that here as well? (It works with my testing.)

```suggestion
            .animation(.default, value: shimmeringActive)
```

Reply:
Yes, good observation 👍